internal/jimm/relation.go

```diff
@@ -110,29 +110,60 @@ func (j *JIMM) ListObjectRelations(ctx context.Context, user *openfga.User, obje
 	if !user.JimmAdmin {
 		return nil, e, errors.E(op, errors.CodeUnauthorized, "unauthorized")
 	}
-	continuationToken, kind, err := pagination.DecodeEntitlementToken(entitlementToken)
+	responseTuples, nextToken, err := j.getObjectRelationsPage(ctx, object, pageSize, entitlementToken)
 	if err != nil {
-		return nil, e, err
+		return nil, e, errors.E(op, err)
 	}
+	// verify next page contains some entries. Otherwise return empty nextToken.
+	if len(responseTuples) == int(pageSize) && nextToken.String() != "" {
+		responseTuples, _, err := j.getObjectRelationsPage(ctx, object, 1, nextToken)
+		if err != nil {
+			return nil, e, errors.E(op, "error getting next page to verify it cointains something", err)
+		}
+		if len(responseTuples) == 0 {
```

Comment:
should we be making this call? or do we trust openfga to know when to stop, by giving us an empty continuation token?